src/Chat/Bot/Cipher.hs

```diff
@@ -0,0 +1,46 @@
+module Chat.Bot.Cipher where
+
+import Chat.Data
+import Data.List
+
+
+-- LEVEL: Easy?
+
+-- Goal is to implement a bot that takes a number and a string
+-- and returns a string with the characters ciphered by RotN 
+-- where N is the integer supplied.
+
+-- There are a lot of ways to do this, pitfalls / PITA issues
+-- if participants take the wrong approach.
+-- key ones I identify:
+-- negative integers (handled by mod in my example bot)
+-- integers greater than 26 (as above)
+-- punctuation
+-- capitalisation
+-- inelgant rotation e.g. excessive type conversions/lookups
+
+-- Below is an example I feel is elegant
+type Shift = Int
+type Phrase = String
+
+cipher :: Char -> [(Char, Char)] -> Char
```

Comment:
I would be tempted to show some "examples" of simple in/out, just to give people an idea of what to expect (and fail if the implementation is incorrect).

```haskell
-- >>> cipher 'a' (cipherTable 1)
-- 'b'
```

I can/will add some for the guess bot and anything else I add.